Frameworks/CoreText/CTFrame.mm

```diff
@@ -78,8 +79,7 @@ CFRange CTFrameGetVisibleStringRange(CTFrameRef frame) {
  @Notes
 */
 CGPathRef CTFrameGetPath(CTFrameRef frame) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return frame ? static_cast<_CTFrame*>(frame)->_path.get() : nil;
```

Comment:
>frame ? static_cast<_CTFrame*>(frame)->_path.get() : nil; [](start = 11, length = 57)

for invalid cases, please use the macros, not only it will support any future alteration, it can also provide more information on the case.
the current one is not so useful when debugging.